a2a/core.go

```diff
@@ -228,6 +244,10 @@ type TaskArtifactUpdateEvent struct {
 	Metadata map[string]any `json:"metadata,omitempty" yaml:"metadata,omitempty" mapstructure:"metadata,omitempty"`
 }
 
+func (a *TaskArtifactUpdateEvent) Meta() map[string]any {
+	return a.Metadata
+}
```

Comment:
Q. This gives two ways of accessing `Metadata` (`a.Metadata` vs `a.Meta()`). Is there a plan to remove one of them (e.g. Metadata)?